e2e/cypress.json

```diff
@@ -2,6 +2,7 @@
     "baseUrl": "http://localhost:8065",
     "defaultCommandTimeout": 20000,
     "ignoreTestFiles": "**/node_modules/**/*",
+    "chromeWebSecurity": false,
```

Comment:
why do we need this?

Reply:
I have used it because we have Cross-origin iframes for displaying a credit card details from Stripe in our subscription screens

Reply:
@saturninoabril do we have any other test that runs into this issue? I feel we shouldn't be calling to stripe in our tests.

Reply:
Our payment form requires the stripe iframe for credit card input and verification so we need to be able to have the tests do this request. We have the same approach in the portal tests setup, see https://github.com/mattermost/customer-web-server/blob/d8fadb8435e3f26ec0f752fc489f7d97f9a882fb/webapp/test/cypress.json#L14

Reply:
@Willyfrog Thanks for checking and as answered above by MN.

Reply:
Please revert this and let it be addressed by the other PR - https://github.com/mattermost/mattermost-webapp/pull/8361/files